subscription-engine/kafka-ws/test/integration/Conditions.test.js

```diff
@@ -19,6 +19,7 @@ const mockMicroServiceSdk = {
     signalReady: jest.fn(),
     signalNotReady: jest.fn(),
     addHealthChecker: jest.fn((service, callback) => callback()),
+    registerShutdownHandler: jest.fn(),
```

Comment:
Do you really need to mock the SDK? I think that many other test files have mocked the SDK without the real need too.